modules/geolocation/outputanalysis.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def get_best_location(array):
+
+    tempArrayOne=array[:,[2]].copy()
+    tempArrayTwo=array[:,[0]].copy()
+    tempArrayThree=array[:,[1]].copy()
+
+#splits the coordinate pairs, error/radius and confidence from the input numpy array into three arrays
+
+    rawArrayOne=[]
+    rawArrayTwo=[]
+    rawArrayThree=[]
+
+    for i in tempArrayOne:
+        for j in list(i):
+            rawArrayOne.append(j)
+    for i in tempArrayTwo:
+        for j in list(i):
+            rawArrayTwo.append(j)
+    for i in tempArrayThree:
+        for j in list(i):
+            rawArrayThree.append(j)
+
+    arrayCoords=np.array(rawArrayTwo)
+    arrayError=np.array(rawArrayThree)
+    arrayConfid=np.array(rawArrayOne)
```

Comment:
Try `np.vstack(array[:, columnNumber]).astype(np.float)`, where `columnNumber` is the number of the column you wish to slice

Also try `np.vstack(array[:,0])` to slice the first column.